packages/flutter/lib/src/material/text_selection_toolbar_text_button.dart

```diff
@@ -0,0 +1,119 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+import 'constants.dart';
+import 'text_button.dart';
+import 'theme.dart';
+
+enum _TextSelectionToolbarItemPosition {
+  /// The first item among multiple in the menu.
+  first,
+
+  /// One of several items, not the first or last.
+  middle,
+
+  /// The last item among multiple in the menu.
+  last,
+
+  /// The only item in the menu.
+  only,
+}
+
+/// A button styled like a Material native Android text selection menu button.
+class TextSelectionToolbarTextButton extends StatelessWidget {
```

Comment:
@HansMuller I'm curious about your opinion on me creating this button, since you just overhauled the button classes.